build/wd_js_bundle.bzl

```diff
@@ -114,8 +115,8 @@ def wd_js_bundle(
         internal_wasm_modules = {},
         internal_data_modules = {},
         declarations = [],
-        **kwargs
-):
+        deps = [],
```

Comment:
let's update the comment.

I highly recommend bazel vscode plugin which will highlight buildifier issues.